Unable to build dotnet/runtime due to cmake error about MultiThreadedDebug

I'm unable to build dotnet/runtime after sync'ing and trying to do a clean build:
```
  CMake Error in D:/repos/runtime/src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt:
    MSVC_RUNTIME_LIBRARY value 'MultiThreadedDebug' not known for this ASM_MASM
    compiler.
```
Note I'm building on Windows; I don't know why it's trying to process a unix cmakelists.

@jkoritzinsky, I assume this is due to https://github.com/dotnet/runtime/pull/41897?

Do I need to somehow update my local configuration?

--- a/eng/native/configurecompiler.cmake
+++ b/eng/native/configurecompiler.cmake
@@ -627,6 +627,10 @@ if (CLR_CMAKE_HOST_WIN32)
       set(CMAKE_ASM_COMPILE_OBJECT "<CMAKE_ASM_COMPILER> -g <INCLUDES> <FLAGS> -o <OBJECT> <SOURCE>")
     else()
       enable_language(ASM_MASM)
+      set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreaded         "")
+      set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreadedDLL      "")
+      set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreadedDebug    "")
+      set(CMAKE_ASM_MASM_COMPILE_OPTIONS_MSVC_RUNTIME_LIBRARY_MultiThreadedDebugDLL "")
     endif()
 
     # Ensure that MC is present

